Description: Link to external json-c.
 Missing link caused build failure :
 .
 /usr/bin/ld: .libs/gdalinfo.o: undefined reference to symbol 'json_object_new_int'
 //lib/x86_64-linux-gnu/libjson-c.so.2: error adding symbols: DSO missing from command line
 collect2: error: ld returned 1 exit status
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://trac.osgeo.org/gdal/ticket/6440

--- a/apps/GNUmakefile
+++ b/apps/GNUmakefile
@@ -30,6 +30,10 @@ ifeq ($(HAVE_GEOS),yes)
 CPPFLAGS 	:=	$(CPPFLAGS) -DHAVE_GEOS=1 $(GEOS_CFLAGS)
 endif
 
+ifeq ($(LIBJSONC_SETTING),external)
+LNK_FLAGS :=	$(LNK_FLAGS) -ljson-c
+endif
+
 NON_DEFAULT_LIST = 	multireadtest$(EXE) dumpoverviews$(EXE) \
 	gdalwarpsimple$(EXE) gdalflattenmask$(EXE) \
 	gdaltorture$(EXE) gdal2ogr$(EXE) test_ogrsf$(EXE) \
